app/javascript/packs/weeatApp/components/restaurantsIcon.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+class RestaurantIcon extends React.Component {
+  render() {
+    const defaultIcon = 'F';
+
+    const cuisineTypeToIcon = {
```

Comment:
This should be inside a table on the server side. If someone wants to add a cuisine he doesn't need to made code changes that way and all his changes are in the same place.

Reply:
moved to DB, will be part of next PR